### PR TITLE
remove google analytics

### PR DESCRIPTION
**Pull request by kkuepper**

### CODE REVIEW

File: client/app/index.html [hunk 1]
```diff
@@ -87,7 +87,6 @@ <h2>We're sorry, it seems like your browser is out of date.</h2>
     <script src="bower_components/angular-ui-bootstrap-bower/ui-bootstrap-tpls.js"></script>
     <script src="bower_components/angular-ui-sortable/sortable.js"></script>
     <script src="bower_components/angulartics/src/angulartics.js"></script>
-    <script src="bower_components/angulartics-google-analytics/lib/angulartics-google-analytics.js"></script>
```
Comment (olivereisenhut) [inline client/app/index.html:90]: There is another script     `<script src="scripts/common/googleanalytics.js"></script>`. It this one still desired for some basic google analytics integration?

**Review by olivereisenhut — APPROVED** [review]
There is still google analytics in the admin site but I think it doesn't hurt anyone there and isn't collecting much data anyway

File: client/app/index.html (hunk 1 above)
Comment (kkuepper) [reply client/app/index.html:90]: We still need it for admin. But thanks for pointing it out. I removed the reference on the website

**Comment by kkuepper** [comment]
> There is still google analytics in the admin site but I think it doesn't hurt anyone there and isn't collecting much data anyway

That's what I thought

**Pull request merged**